src/main/java/net/jodah/typetools/TypeResolver.java

```diff
@@ -178,26 +184,34 @@ public static void disableCache() {
    */
   public static Type resolveGenericType(Class<?> type, Type subType) {
     Class<?> rawType;
-    if (subType instanceof ParameterizedType)
+    if (subType instanceof ParameterizedType) {
       rawType = (Class<?>) ((ParameterizedType) subType).getRawType();
-    else
+    } else {
       rawType = (Class<?>) subType;
+    }
 
-    if (type.equals(rawType))
+    if (type.equals(rawType)) {
       return subType;
+    }
 
     Type result;
     if (type.isInterface()) {
-      for (Type superInterface : rawType.getGenericInterfaces())
-        if (superInterface != null && !superInterface.equals(Object.class))
-          if ((result = resolveGenericType(type, superInterface)) != null)
+      for (Type superInterface : rawType.getGenericInterfaces()) {
+        if (superInterface != null && !superInterface.equals(Object.class)) {
```

Comment:
It looks like most of this change is adding braces where there are none currently. I'd prefer to keep the control statements as they are. Can you update the commit?